handlers/TileMapHandler.js

```diff
@@ -244,6 +246,44 @@ class TileMapHandler {
         }
     }
 
+    resetPlayerStartPoint() {
+        const levelObjects = this.levelObjects;
+        const startFlagsInLevel = levelObjects.filter(levelObject => levelObject.type === ObjectTypes.START_FLAG);
+        const doorsInLevel = levelObjects.filter(levelObject => levelObject.type === ObjectTypes.DOOR);
```

Comment:
you could combine those filters into 1 array. if type === ObjectTyptes.START_FLAG || type === ObjectTypes.DOOR
as far as i've seen you don't need 2 separate arrays

Reply:
Yep, this is WIP stuff so the two different objects can be handled differently as I test out UI variations. Honestly at this stage though I think they'll just end up getting handled identically. They will most likely be merged together in the final PR.